doc/man/po/es.po

```diff
@@ -1073,18 +1073,18 @@ msgstr "MAS INFORMACIÓN"
 #. type: textblock
 #: darktable.pod:498
 msgid ""
-"Please visit B<darktable>'s website for news, blog and bug tracker: L<http://"
+"Please visit B<darktable>'s website for news, blog and bug tracker: L<https://"
```

Comment:
Please don't touch .po files.

Reply:
but why, are they auto-generated? (I'm not the author, just curious)

Reply:
git conflict resolution is no fun even for knowledged developers.
and usually tranlators are less knowledged than the developers.
so for them it might mean "drop all your local changes and re-translate from the scratch"